Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/configuration.py

```diff
@@ -0,0 +1,562 @@
+config = {
```

Comment:
Maybe it would be simpler for the users to use the one that I will put in the 2 qubit scripts without the '$' signs and specificities to the KIT chip and research team code.
Are names from the config are coded somewhere in the 2qb RB functions?